msg/vehicle_status.msg

```diff
@@ -49,6 +49,11 @@ uint8 RC_IN_MODE_DEFAULT = 0
 uint8 RC_IN_MODE_OFF = 1
 uint8 RC_IN_MODE_GENERATED = 2
 
+uint8 VEHICLE_TYPE_UNKNOWN = 0
+uint8 VEHICLE_TYPE_ROTARY_WING = 1
+uint8 VEHICLE_TYPE_FIXED_WING = 2
+uint8 VEHICLE_TYPE_GROUND = 3
```

Comment:
What about `VEHICLE_TYPE_ROVER`? What was your idea behind `GROUND`?

Reply:
I don't know if there will ever be a ground vehicle that's not called a rover, so I opted for the most generic name. But if that doesn't make sense, it should be an easy change to just call it `VEHICLE_TYPE_ROVER`.

Reply:
We should decide one way or the other and try to be consistent.

Reply:
Options
1. Go with rover, but all update all mentions in init scripts, mixers, controllers, documentation, etc.
2. Continue with UGV, but then this constant becomes `uint8 VEHICLE_TYPE_UNMANNED_GROUND_VEHICLE = 3` (verbose and redundant). 

If I had to choose I'd go with 1, but ultimately I just want consistency.

Reply:
Why `UNMANNED`? We never know :smile: ...

Reply:
I have changed it to `VEHICLE_TYPE_ROVER`, and updated all mentions of `ugv` to `rover` throughout the init scripts.